Sources/XMTP/Client.swift

```diff
@@ -47,7 +51,7 @@ public class Client {
 		let privateKeyBundleV1 = try await loadOrCreateKeys(for: account, apiClient: apiClient)
 
 		let client = try Client(address: account.address, privateKeyBundleV1: privateKeyBundleV1, apiClient: apiClient)
-		try await client.publishUserContact()
+		try await client.ensureUserContactPublished()
```

Comment:
Brought over the logic to not constantly be publishing new contact bundles here.